elasticsearch/Dockerfile

```diff
@@ -30,7 +30,8 @@ RUN packages="java-${JAVA_VER}-openjdk-headless \
               hostname \
               openssl \
               zip \
-              unzip" && \
+              unzip \
+              jq" && \
```

Comment:
I thought we were not recommended to pull in `jq`?

Reply:
not that i know of... ART has built it for other images

Reply:
> I thought we were not recommended to pull in `jq`?

I believe this was only because it was not available but has since been productized